homework/week-0/src/nyc/c4q/JVidals1/DeckofCards.txt

```diff
@@ -0,0 +1,83 @@
+package nyc.c4q.JVidals1;
+import javax.smartcardio.Card;
+
+/**
+ * Created by s3a on 5/1/15.
+ * Janneisy Vidals
+ * 2.1 Accesscode
+ * HW Excercises
+ * Description:
+ * Due March 27, 2015.
+ */
+
+
+public abstract class DeckofCards
+{
+    public static final int NumOfCards = 52;
+
```

Comment:
Lines 45 & 46--great way to do brief comments at the end of each line to explain what's going on there.